be/src/vec/exec/format/json/new_json_reader.cpp

```diff
@@ -1370,53 +1374,40 @@ Status NewJsonReader::_simdjson_handle_nested_complex_json(
     return Status::OK();
 }
 
-size_t NewJsonReader::_column_index(const StringRef& name, size_t key_index) {
```

Comment:
this function accelerate parsing speed, do not delete it

Reply:
OK，I will add it back

Reply:
> this function accelerate parsing speed, do not delete it

I think this method is used to get the index of a column in a block, but if we iterate `slot_descs` through `for (size_t i = 0; i < slot_descs.size(); ++i)` instead of iterating `*value` through `for (auto field : *value)`, we can get the index `i` directly whether this method can be deleted or not.

Reply:
> this function accelerate parsing speed, do not delete it

I have added it back